bindings_generator/src/class_docs.rs

```diff
@@ -84,19 +84,46 @@ impl GodotXMLDocs {
 
     fn parse_method(&mut self, class: &str, method: Node) {
         if let Some(method_name) = method.attribute("name") {
+            let default_args = method
+                .descendants()
+                .filter_map(|node| {
+                    if node.tag_name().name() == "argument" && node.has_attribute("default") {
+                        let val = node.attribute("default").unwrap();
+                        let key = node.attribute("name").unwrap();
+                        Some((key, val))
+                    } else {
+                        None
+                    }
+                })
+                .collect::<Vec<(&str, &str)>>();
+
             if let Some(desc_node) = method
                 .descendants()
                 .find(|node| node.tag_name().name() == "description")
             {
                 if let Some(desc) = desc_node.text() {
-                    self.add_fn(class, method_name, desc);
+                    self.add_fn(class, method_name, desc, default_args.as_slice());
                 }
             }
         }
     }
 
-    fn add_fn(&mut self, class: &str, method: &str, desc: &str) {
+    fn add_fn(&mut self, class: &str, method: &str, desc: &str, default_args: &[(&str, &str)]) {
+        let mut doc = desc.trim().replace("\n\n", "\n");
+
+        if doc.is_empty() {
+            return;
+        }
```

Comment:
Possible logic error: default arguments won't be documented if the method doesn't have a description.

Reply:
ok it now only skips if the default args are also empty